### PR TITLE
Add test for change in pull request #1631

### PR DESCRIPTION
**Pull request by obestwalter**
Add test for the changed assertion as suggested by @The-Compiler

### CODE REVIEW

File: testing/python/fixture.py [hunk 1]
```diff
@@ -490,6 +490,21 @@ def test_func(something): pass
...
+
+            @pytest.fixture
+            def pytest_funcarg__marked_with_prefix_and_decorator():
+                pass
+        """)
+        result = testdir.runpytest_subprocess()
+        assert result.ret != 0
+        result.stdout.fnmatch_lines([
+            "*AssertionError: *",
+            "*pytest_funcarg__marked_with_prefix_and_decorator*"
```
Comment (blueyed) [inline testing/python/fixture.py]: IIRC the first line will eat the line itself, and the error looks like: `AssertionError: pytest_funcarg__marked_with_prefix_and_decorator`.

Just match that line (as a single line) instead.

Comment (obestwalter) [inline testing/python/fixture.py]: Checking for both occurrences works as expected. Also just checking for each single occurrence. If I change the check to :

>         result.stdout.fnmatch_lines([
>           "*AssertionError*pytest_funcarg__marked_with_prefix_and_decorator*"
>         ])
>     E       Failed: remains unmatched: 
>     '*AssertionError*pytest_funcarg__marked_with_prefix_and_decorator*', see stderr

So I guess it's o.k. to explicitly check first for the assertion error and then for the wanted function name?

**Comment by obestwalter** [comment]
As far as I see those failures are unrelated to my change, but I will check tomorrow.

File: testing/python/fixture.py (hunk 1 above)
Comment (nicoddemus) [inline testing/python/fixture.py]: I'm not sure I follow, but the order here matters, I guess that's what @blueyed mentioned

Comment (nicoddemus) [inline testing/python/fixture.py]: To be honest we should have a better message than just the function name... perhaps:

```
pytest_funcarg__marked_with_prefix_and_decorator is also decorated with @pytest.fixture; 
It is recommended to remove the "pytest_funcarg__" prefix.
```

**Comment by obestwalter** [comment]
Thanks @pelme and @blueyed for helping me to get started with testing pytest with pytest and getting my first pytest test into pytest :)

**Comment by pelme** [comment]
Very nice 👍 ⚡ 😃

**Comment by The-Compiler** [comment]
And thank you for the follow-up!

**Pull request merged**